name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/OnvistaPDFExtractor.java

```diff
@@ -499,11 +499,27 @@ private void addDividendeTransaction()
 
                 .oneOf(
                             // STK 50,000 21.04.2016 21.04.2016 EUR 0,200000
+                            
+                            section -> section
+                                    .attributes("date","exDate")
+                                    .match("^STK [\\.,\\d]+ (?<exDate>[\\d]{2}\\.[\\d]{2}\\.[\\d]{4})? (?<date>[\\d]{2}\\.[\\d]{2}\\.[\\d]{4}) [\\w]{3} [\\.,\\d]+$")
```

Comment:
remove the `?` after `(?<exDate>[\\d]{2}\\.[\\d]{2}\\.[\\d]{4})`

Reply:
changed